Assets/Mirror/Runtime/NetworkStartPosition.cs

```diff
@@ -11,19 +11,5 @@ namespace Mirror
     [HelpURL("https://mirror-networking.com/docs/Components/NetworkStartPosition.html")]
     public class NetworkStartPosition : MonoBehaviour
```

Comment:
Why do you keep this empty script? NetworkManager is using transform, so you can safely remove it and use Transform instead